templates/chatbots/settings_content.html

```diff
@@ -0,0 +1,264 @@
+{% load form_tags %}
+{% if updated %}
+  <div id="chatbot-name" hx-swap-oob="innerHTML">{{ experiment.name }}</div>
+  <div id="chatbot-description" hx-swap-oob="innerHTML">{{ experiment.description }}</div>
+{% endif %}
+<div id="settings-wrapper">
+  <div class="flex justify-end mb-6">
+    {% if edit_mode %}
+      <div class="flex gap-2">
+        <button id="save-all-button"
+                class="btn btn-success btn-sm"
+                hx-post="{% url 'chatbots:settings' request.team.slug experiment.id %}"
+                hx-include="#settings-form"
+                hx-target="#settings-wrapper"
+                hx-swap="outerHTML">
+          Save
+        </button>
+        <button id="cancel-edit-button"
+                class="btn btn-danger btn-sm"
+                hx-get="{% url 'chatbots:cancel_edit_mode' request.team.slug experiment.id %}"
+                hx-target="#settings-wrapper"
+                hx-swap="outerHTML">
+          Cancel
+        </button>
+      </div>
+    {% else %}
+      <div id="edit-buttons-container">
+        {% if experiment.is_editable %}
+          <button id="edit-all-button"
+                  class="btn btn-primary btn-sm"
+                  hx-get="{% url 'chatbots:settings' request.team.slug experiment.id %}"
+                  hx-target="#settings-wrapper"
+                  hx-swap="outerHTML">
+            Edit
+          </button>
+        {% endif %}
+      </div>
+    {% endif %}
+  </div>
+
+  {% if edit_mode %}
+    <div style="display: none;">
+      <div id="voiceProviderTypes">{{ voice_providers_types|safe }}</div>
+      <div id="voiceOptions">{{ synthetic_voice_options|safe }}</div>
+    </div>
```

Comment:
Rather than using a div here you could use the `json_script` tag which is designed for doing exactly this: https://docs.djangoproject.com/en/5.2/ref/templates/builtins/#json-script

It would also mean you don't have to convert the data to JSON first, a normal dict will be fine.